grading-container/script/grading_job/grading_job_builder.py

```diff
@@ -14,7 +15,7 @@ class GradingJobBuilder:
   def __init__(self) -> None:
```

Comment:
See comments above.  If the arguments to `create_grading_job` are required, then make them arguments to the constructor.